client/src/containers/CampaignAdmin/tests/CampaignAdmin-test.js

```diff
@@ -0,0 +1,82 @@
+/* global jest, describe, beforeEach, it, expect */
+
+jest.mock('react-bootstrap-ss');
+jest.mock('lib/Backend');
+
+import { CampaignAdminBase } from '../CampaignAdmin';
+
+describe('CampaignAdminItem', () => {
+  describe('detectErrors', () => {
+    const admin = new CampaignAdminBase({
```

Comment:
protect unit tests from mutations where possible, so this line could be rewritten as:
```
let admin = null;

beforeEach(() => {
  admin = new CampaignAdminBase({
```